web/templates/user_group_settings/user_group_settings_overlay.hbs

```diff
@@ -42,7 +47,7 @@
                         </span>
                     </div>
                 </div>
-                <div class="user-groups-list" data-simplebar data-simplebar-tab-index="-1">
+                <div class="user-groups-list {{#if deactivated}}hide-deactived-user-groups{{/if}}" data-simplebar data-simplebar-tab-index="-1">
```

Comment:
I don't see `deactivated` being passed as argument while rendering this. Or maybe I am missing something.